manifests/charts/gateway/values.yaml

```diff
@@ -139,3 +139,8 @@ volumes: []
 # A list of `VolumeMounts` added into the Gateway Pods. See
 # https://kubernetes.io/docs/concepts/storage/volumes/.
 volumeMounts: []
+
+# Platform where Istio is deployed. Possible values are: "openshift", "gcp".
+# An empty value means it is a vanilla Kubernetes distribution, therefore no special
+# treatment will be considered.
+platform: ""
```

Comment:
Yeah again as mentioned previously in Slack - having a "platform" value that implies $some-arbitrary-set-of-other-values is far less obvious and flexible in the long term than just putting platform-specific values overrides a specific platform values file.

The values overrides a given platform needs should be 

1. All in a single file
2. Easy to add to or subtract from or alter *without* making template changes every time.

e.g. if say GCP needs a new override that Openshift already needs/requires or vice-versa - we shouldn't need to update a conditional tree in the templates for that. If we do we're sort of intentionally defeating how Helm and values overrides are designed to be used.

`openshift-values.yaml` > `--set values.platform=openshift`